version bump for NodeJS to 16

corporate-gadfly opened the pull request:
- fixes #28

corporate-gadfly committed:
version bump for NodeJS to 16

- fixes #28

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/aquasecurity/tfsec-azure-pipelines-task?pullRequest=29) <br/>All committers have signed the CLA.

corporate-gadfly committed:
migrate task to Node 16

follow [migration guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode16.md)

segraef commented:
Any updates on this one?
